### ISSUE TITLE
why the auc of the ped1 is very low (about 50%)?

### ISSUE THREAD

**Issue by KingMV** [tnybny/Frame-level-anomalies-in-videos#1]
Dear tnybny, Thank you very much for your code about abnormal video detection ! when I use your code to train STAE model ,the auc about the  dataset of ped1 is very low (about 50% ). so ,I want to know your results about your model? Thanks!

**Comment by tnybny** [comment]
@KingMV This code is still under active development. However, you should be able to get an AUC of 0.73 easily. From my experience, when AUC is as low as 0.5, there is some error during data preprocessing (either data or labels). Make sure you use src/create_datasets.py to preprocess your data.

**Comment by KingMV** [comment]
@tnybny Thanks for your replys, I have trained the CONVAE model,I find the best AUC is 0.72.But I find that the loss is very hight (about 54796) when the iterations than 12000.Can you tell me some reasons?Thanks!

**Comment by tnybny** [comment]
@KingMV I should tell you that I'm tinkering with CONVAE model in the latest commits. I believe commit f561be5d4375400d5270a65f405966e1ef606597 is a stable version for CONVAE. A loss that high at the end of training is unexpected - you should look at the loss curve to see if it decreases at all.

**Comment by youngupup** [comment]
you can give me the code of compute the frame auc ?thanks

**Comment by tnybny** [comment]
Hi, for frame level auc you can just use https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#sklearn.metrics.roc_auc_score. Please open a new issue in the future for better communication.

**Comment by youngupup** [comment]
> Hi, for frame level auc you can just use https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#sklearn.metrics.roc_auc_score. Please open a new issue in the future for better communication.

Thanks！！！
